src/sentry/static/sentry/app/views/admin/adminOrganizations.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import {Link, RouteComponentProps} from 'react-router';
+
+import ResultGrid from 'app/components/resultGrid';
+import {t} from 'app/locale';
+
+type Props = RouteComponentProps<{}, {}>;
+
+const getRow = (row: any) => [
+  <td key={row.id}>
+    <strong>
+      <Link to={`/${row.slug}/`}>{row.name}</Link>
+    </strong>
+    <br />
+    <small>{row.slug}</small>
+  </td>,
+];
+
+const AdminOrganizations = (props: Props) => (
+  <div>
```

Comment:
would be fine to replace the `div` with ` React.Fragment`?

Reply:
maybe 😬 not gonna look to hard since this is just an admin view